app/main.js

```diff
@@ -179,6 +181,24 @@ function createWindow() {
         }
     })
 
+    storage.has('settings', function(error, hasKey) {
+        if (error) {
+            console.log('d1')
+            electron.nativeTheme.themeSource = 'dark'
+        };
+      
+        if (hasKey) {
+            storage.get('settings', function(error, object) {
+                if (error) {
+                    console.log('d2')
+                    electron.nativeTheme.themeSource = 'dark'
+                } else {
+                    electron.nativeTheme.themeSource = object.theme;
+                }
+            });
+        }
```

Comment:
Some things I just noticed:

- Anonymous functions should be defined with arrow syntax preferably. So

  ```js
  storage.get('settings', function(error, object) {
      // ...
  }
  ```

  becomes

  ```js
  storage.get('settings', (error, object) => {
      // ...
  }
  ```

- Please write something meaningful in the log messages, e.g. the error object itself.
- Omitting semicolons is preferred in this project. The semicolon in line 188 is even not quite right.

Reply:
Completed!